ui/packages/atlasmap/src/impl/toolbarItems.tsx

```diff
@@ -113,16 +114,16 @@ export const AtlasmapToolbarItem: FunctionComponent<
     showExportAtlasFileToolbarItem && (
       <DropdownSeparator key="export-separator" />
     ),
-    false /* showDeleteResetToolbarItem #3866 */ && (
+    showDeleteResetToolbarItem && (
       <DeleteDocsAndMappingsToolbarItem
         docsExist={docsExist!}
         onClick={runAndClose(onDeleteDocsAndMappings)}
         key="del-docs-mappings"
       />
     ),
-    false /* showDeleteResetToolbarItem #3867 */ && (
+    showDeleteResetToolbarItem && (
       <DeleteLibrariesToolbarItem
-        librariesExist={librariesExist!}
+        userLibrariesExist={librariesExist!}
```

Comment:
Can you either 1) keep this disabled or 2) address stale mappings and Documents in https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/main/java/io/atlasmap/service/AtlasService.java#L294